inc/saiport.h

```diff
@@ -2520,6 +2520,14 @@ typedef enum _sai_port_attr_t
      */
     SAI_PORT_ATTR_POE_PORT_ID,
 
+    /**
+     * @brief Fabric port reachability for all remote reachable switches
```

Comment:
Description is not exactly saying what is the value of the list

Reply:
Fixed